packages/flutter_test/lib/src/controller.dart

```diff
@@ -242,23 +242,27 @@ class SemanticsController {
   /// * [flutter/engine/AccessibilityBridge.java#SemanticsNode.isFocusable()](https://github.com/flutter/engine/blob/main/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L2641)
   /// * [flutter/engine/SemanticsObject.mm#SemanticsObject.isAccessibilityElement](https://github.com/flutter/engine/blob/main/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm#L449)
   bool _isImportantForAccessibility(SemanticsNode node) {
+    if (node.isMergedIntoParent) {
```

Comment:
Maybe just leave a quick comment here for context that if the node is merged, all its information are present on an ancestor node.

Reply:
done